wb-mqtt-snmp.schema.json

```
@@ -193,11 +193,66 @@
     "max_unchanged_interval": {
       "type": "integer",
       "title": "Unchanged value posting interval",
-      "description": "Maximum interval in seconds between posting the same value to message queue. Zero - post at every reading, negative - don't post the same values",
+      "description": "max_unchanged_interval_description",
       "default": -1,
       "propertyOrder": 30
     }
   },
 
-  "required": [ "devices" ]
+  "required": [ "devices" ],
+
+  "translations": {
+    "en": {
+      "snmp_title": "SNMP Device Driver Configuration",
+      "snmp_description": "List devices to poll via SNMP protocol",
+      "oid_prefix_description": "Common prefix for names in channel OIDs (for SNMPv2-MIB::sysLocation.0 prefix is SNMPv2-MIB). May be overriden by writing OID in channel with prefix and '::'",
+      "poll_interval_description": "This option specifies the desired interval between two consecutive polls of each channel (or total duration of the poll cycle)",
+      "channels_description": "List device variables and their corresponding controls",
+      "units_description": "Value units of measure (V, A, kWh etc.). Only for control_type == 'value'",
+      "max_unchanged_interval_description": "Maximum interval in seconds between posting the same value to message queue. Zero - post at every reading, negative - don't post the same values"
+    },
+    "ru": {
+      "snmp_title": "Настройка драйвера SNMP-устройств",
+      "snmp_description": "Список устройств для опроса по протоколу SNMP",
+      "SNMP device": "SNMP устройство",
+      "Enable device": "Включить устройство",
```

Кажется, что эта галочка физически включит устройство, а на самом деле включит только его опрос.
```suggestion
      "Enable device": "Опрашивать устройство",
```

или "Активно"